sklearn/ensemble/tests/test_bagging.py

```diff
@@ -748,3 +749,99 @@ def test_set_oob_score_label_encoding():
     x3 = BaggingClassifier(oob_score=True,
                            random_state=random_state).fit(X, Y3).oob_score_
     assert_equal([x1, x2], [x3, x3])
+
+
+def test_bagging_regressor_with_missing_inputs_and_1d_output():
+    # Check that BaggingRegressor can accept X with missing/infinite data
+    X = [
+        [1, 3, 5],
+        [2, None, 6],
+        [2, np.nan, 6],
+        [2, np.inf, 6],
+        [2, np.NINF, 6],
+    ]
+    Y = [2, 3, 3, 3, 3]
+    regressor = DecisionTreeRegressor()
+    pipeline = make_pipeline(
+        Imputer(),
+        Imputer(missing_values=np.inf),
+        Imputer(missing_values=np.NINF),
+        regressor
+    )
+    pipeline.fit(X, Y).predict(X)
+    bagging_regressor = BaggingRegressor(pipeline)
+    bagging_regressor.fit(X, Y).predict(X)
```

Comment:
should check decision_function etc also, no?

Reply:
There is no `decision_function` on `BaggingRegressor`.
I'll add it to the `BaggingClassifier` test.

Reply:
I actually can't add it to the BaggingClassifier because DecisionTreeRegressor does not support decision_function.